app/views/comment_mailer/comment_reply_sent_notification.html.erb

```diff
@@ -0,0 +1,33 @@
+<% content_for :message do %>
+
+  You replied to a comment on
+  <% if @comment.ultimate_parent.is_a?(Tag) %>
+    the tag
+      <%= style_link(@comment.ultimate_parent.commentable_name, tag_url(@comment.ultimate_parent)) %>:
+  <% else %>
+    <i><b><%= style_link(@comment.ultimate_parent.commentable_name.html_safe, polymorphic_url(@comment.ultimate_parent, only_path: false)) %></b></i>:
+  <% end %>
+
+  <% commenter = @parent_comment.comment_owner ?
+      (
+        (@parent_comment.comment_owner.is_author_of?(@parent_comment.ultimate_parent) &&
```

Comment:
Since #4237 you can simplify this check to `@parent_comment.by_anonymous_creator?`. Same for the text email.